manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Statsify.StatsifyOverlay
+PackageVersion: 2.7.2
+PackageLocale: en-US
+Publisher: Statsify
+PackageName: Statsify Overlay
+License: GNU License
+ShortDescription: Statsify Overlay

```

```suggestion
ShortDescription: Statsify Overlay for Hypixel Bedwars
Description: 
```
Can you add a better ShortDescription and Description?